test/fw/ptl/utils/pbs_snaputils.py

```diff
@@ -818,6 +799,8 @@ def __capture_trace_from_core(self, core_file_name, exec_path, out_path):
         gdb.execute("quit")
         quit()
         """ % (core_file_name)
+        # Remove tabs from triple quoted strings
+        gdb_python = gdb_python.replace("\t", "")
```

Comment:
It would look weird without the indent. The other option is to create a module level constant, but this is not really a constant since it is a formatted string, so I didn't wanna do that either. What do you think?